README.md

```diff
@@ -1,33 +1,81 @@
-# Project
+# Bicep REPs
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+A Bicep REP (Request for Enhancement Proposal) is a formal way to suggest significant changes to Bicep. It offers the Bicep team and the community an organized platform to share feedback, ensuring consensus on the project's direction.
 
-As the maintainer of this project, please make a few updates:
+## When to Follow the Process
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Consider employing this process if you intend to make substantial changes to the Bicep programming language that would affect user experience. Example where the use of a REP is warranted include:
+- *Introduction of New Features*: If the proposed change involves the addition of a new syntax kind or a new ARM template function, particularly if it necessitates the use of a feature flag.
```

Comment:
"feature" is pretty broad - do we specifically mean a syntax change?

Reply:
Improved this section and added a few more examples.